web/controllers/session_controller.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Community.SessionController do
+  use Community.Web, :controller
+
+  def new(conn, _params) do
+    conn
+    |> render(:new)
```

Comment:
I considered it but there is a good chance this will include more in the future so I think I'd prefer to keep it split.